packages/fxa-settings/src/components/Settings/FlowRecoveryKeyConfirmPwd/index.test.tsx

```diff
@@ -215,11 +221,164 @@ describe('FlowRecoveryKeyConfirmPwd', () => {
       screen.getByText('Unexpected error');
       expect(logViewEvent).toBeCalledWith(
         `flow.${viewName}`,
-        'confirm-password.submit'
+        'create-key.confirm-pwd.submit'
       );
       expect(logViewEvent).toBeCalledWith(
         `flow.${viewName}`,
-        'confirm-password.fail'
+        'create-key.confirm-pwd.fail'
+      );
+      expect(navigateForward).not.toBeCalled();
+    });
+  });
+
+  // TODO expect metric event when back arrow clicked
```

Comment:
Do you think we would want to have a metric event that matched the existing "navigate-forward" pattern of like, `{stepName}.navigate-backward`, or just 'navigate-to-settings', or something else entirely?

Reply:
I've updated with a suggested metric of `create-key.cancel`/`change-key.cancel` - in terms of user journey, clicking the back arrow at this point of the flow will cancel/abort the key generation. How does that sound?